tests/src/test/scala/cats/tests/FunctorSuite.scala

```diff
@@ -37,4 +37,12 @@ class FunctorSuite extends CatsSuite {
       assert(widened eq list)
     }
   }
+
+  test("ifF equals map(if(_) ifTrue else ifFalse)") {
```

Comment:
That tests ensures the implementation is equivalent to the default one, but doesn't really check whether the default one works. I'd suggest adding some tests with concrete lists and concrete expected values.

Reply:
Tests were added